app/javascript/components/GuitarInfo.js

```diff
@@ -7,10 +8,61 @@ import {
   Card,
   CardTitle,
   CardBody,
-  CardFooter
+  CardFooter,
+  Button,
+  Modal,
+  ModalVariant,
+  Form,
+  FormGroup,
+  TextInput
 } from '@patternfly/react-core';
+import { getGuitarUrlWithId, OPEN, CLOSE } from './constants.js';
+
+const GuitarInfo = ({ id, name, url, price, description }) => {
+  const [isFormModalOpen, setFormModal] = useState(CLOSE);
+
+  const openFormModal = () => {
+    setFormModal(OPEN);
```

Comment:
I think now it is better, when we explicitly mention: 'OPEN' & 'CLOSE'
WYT @MariaAga ?